jobs/build_docker/docker_post_test.groovy

```diff
@@ -74,6 +74,10 @@ def generateTestBranches(function_test){
                         }
                     }
                 } finally{
+                    // Clean up test stack
+                    sh '''#!/bin/bash -x
+                    ./build-config/jobs/FunctionTest/cleanup.sh
+                    '''
```

Comment:
This ```finally``` is outside the node, so the ```cleanup.sh``` won't clean up the dirty space of the node which runs test.